.github/workflows/main-build.yml

```
@@ -40,7 +40,7 @@ jobs:
 
       - name: Set up Docker Buildx
         if: (github.ref == 'refs/heads/main') || (contains(github.event.pull_request.labels.*.name, 'push-image')) || (contains(github.event.pull_request.labels.*.name, 'devnet-test-e2e'))
-        uses: docker/setup-buildx-action@v3
+        uses: docker/setup-buildx-action@v4
```

v4 of `docker/setup-buildx-action` doesn't exist: https://github.com/docker/setup-buildx-action/releases

```suggestion
        uses: docker/setup-buildx-action@v3
```